### DISCUSSION TITLE
Is it possible to adjust meta tags (title) for a nested error boundary?

### DISCUSSION THREAD

**Discussion by mattoni** [remix-run/remix#2101] [Q&A] [answered]
If I've got a nested error boundary, the root renders the <html />, <body /> etc and the route has its own meta function export. Is there a way to provide a custom meta function for when there is an error?

Thanks!

**Comment by sergiodxa** [comment, marked as answer]
The MetaFunction of the nested route will still run, so you can detect there if it was an error (it doesn't have data) and change the title.
